docs/content/guides/developer/advanced/move-2024-migration.mdx

```diff
@@ -40,7 +40,7 @@ Use the new [VSCode Move extension](https://marketplace.visualstudio.com/items?i
 cargo install --git https://github.com/MystenLabs/sui.git move-analyzer
 ```
 
-See the getting started guide on [Move IDEs and plugins](../getting-started/sui-environment.mdx#move-ides-and-plugins) for more information.
+See the getting started guide on [Move IDEs and plugins](../../../references/contribute/sui-environment.mdx#move-ides-and-plugins) for more information.
```

Comment:
Move IDEs and Plugins should stay in the Getting Started guide, and not the contribution guide.